pkg/noun/events.c

```diff
@@ -1004,25 +1004,28 @@ _ce_backup(void)
 
   snprintf(ful_c, 8192, "%s/.urb/bhk", u3P.dir_c);
 
-  if ( c3_mkdir(ful_c, 0700) ) {
+  if ( (c3n == ovw_o) && c3_mkdir(ful_c, 0700) ) {
     if ( EEXIST != errno ) {
       fprintf(stderr, "loom: image backup: %s\r\n", strerror(errno));
+      fprintf(stderr, "loom: image backup failed\r\n");
+      return c3n;
     }
-    return;
```

Comment:
This change is not right -- we need to return in the outer conditional. Also, no additional printf is needed.